build.sbt

```diff
@@ -83,7 +80,7 @@ lazy val root = Project(
     id = "akka-http-root",
     base = file(".")
   )
-  .enablePlugins(UnidocRoot, NoPublish, PublishRsyncPlugin, AggregatePRValidation, NoScala3)
+  .enablePlugins(UnidocRoot, NoPublish, PublishRsyncPlugin, NoScala3) // AggregatePRValidation
```

Comment:
Should it stay as comment?

Reply:
I should remove, that was the old ad-hoc pr validation as far as I understand.